src/khoj/processor/conversation/prompts.py

```diff
@@ -491,6 +499,35 @@
 """.strip()
 )
 
+# Schedule task
+# --
+crontime_prompt = PromptTemplate.from_template(
+    """
+You are Khoj, an extremely smart and helpful task scheduling assistant
+- Given a user query, you infer the date, time to run the query at as a cronjob time string
+- The cron job should run in {server_timezone} time zone
+- Infer user's time zone from the current location provided in their message
```

Comment:
For Whatsapp users:
- Using their phone number to infer timezone maybe a good default?
- We can allow folks to configure their location, scheduled tasks via a /configure command or Whatsapp form UI at some point?
- We can't message after 24 hours since user messaged Khoj on Whatsapp, right? If so, maybe we should warn them of that when they create a scheduled task

For VPN users:
- The degradation in experience due to incorrect inferred location already exists for Khoj users. To mitigate this we should allow folks to explicitly configure their location in their settings. Khoj should use that location instead of trying to infer location based on I.P if it is explicitly set by users

In my opinion, these are good follow-on tasks to tackle after the MVP of scheduled tasks is out. What do you think?

Reply:
I'm merging this PR into it's feature branch but please feel free to comment here or in the Scheduled Tasks Prompt improvement PR at #718